### DISCUSSION TITLE
ContractKit sending multiple transactions to Valora

### DISCUSSION THREAD

**Discussion by fbaud** [celo-org/celo-monorepo#9907] [Q&A]
I'm using ContractKit 1.5.2 and have an action generating multiple atomic transactions that are sent to Valora for acceptation. Currently, after accepting the initial connection with my Dapp, the Valora user has to accept each transaction, one after the other, which is tedious and quite unsettling.

I'd like to know if there is a possibility to ask Valora to accept all subsequent transactions from a walletconnect client, provided that it is staying below a certain total amount accepted by the user for all subsequent transactions?

**Comment by arthurgousset** [comment]
Hmm hadn't heard of accepting transactions on Valora before 🤔 Perhaps this is a question @jeanregisser can speak to?

**Reply by fbaud** [reply]
@0xarthurxyz thanks for the rebound! 

I've added this subject as a proposal for next "Celo Wallet and (d)App Council". I'm currently working on a project where interactions between the dapp and the wallet may be quite intensive and it would definitely require cooperation between the different implementations to deliver a seamless user experience.

**Reply by arthurgousset** [reply]
Sounds good, chat next Wednesday!
